### PR TITLE
Display graphic of total points and activities logged

### PR DESCRIPTION
**Pull request by Chris-Maina**
##### What does this PR do?
Add graphic of logged in user, total points and activities logged.
##### Description of Task to be completed?
* Add `fetchUserActivities` saga to get points earned and activities logged of a user
* Add the above data to store
* Display the above data on Dashboard Component

##### Background context
This branch has changes from this [PR](https://github.com/andela/andela-societies-frontend/pull/178)

##### What are the relevant Pivotal Tracker stories?
[163083347](https://www.pivotaltracker.com/story/show/163083347)

##### Screenshots?
screen shot 2019-02-11 at 11 23 22 am

### CODE REVIEW

File: src/app/Dashboard/components/DashboardComponent.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    fetchUserActivites(userInfo.id);
+  }
+
+  render() {
+    const {
+      error, loading, pointsEarned, activitiesLogged,
+    } = this.props;
+    const { user } = this.state;
+    if (loading) return <p>Loading ...</p>;
+    if (!loading && error) return <p>The was an error while fetching your data. Please try again later.</p>;
```
Comment (kimani-ndegwa) [inline src/app/Dashboard/components/DashboardComponent.js]: We have talked about this before. Please evaluate the right JSX before rendering the element that you would like tp render.

File: src/app/Dashboard/components/tests/DashboardComponent.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { DashboardComponent } from '../DashboardComponent';
+
+describe('<DashboardComponent />', () => {
+  const setUpWrapper = ({
+    error = null,
+    loading = false,
+    pointsEarned = 0,
+    userActivities = [],
+    activitiesLogged = 0,
+  } = {}) => {
```
Comment (kimani-ndegwa) [inline src/app/Dashboard/components/tests/DashboardComponent.test.js]: What does line 6 to line 13 do. I can see the destructure with defaults but other than that, it is not very readable.

Comment (Chris-Maina) [reply src/app/Dashboard/components/tests/DashboardComponent.test.js]: Line 6 is a function that returns the shallow/mounted component. Line 13, maps parameters passed  to props. The function helps to change props passed to the shallow/mounted component. Such as in line `27, 32`

File: src/app/Dashboard/components/DashboardComponent.js (hunk 1 above)
Comment (Chris-Maina) [reply src/app/Dashboard/components/DashboardComponent.js]: Will change that in a few.

**Pull request merged**